### ISSUE TITLE
When auditing two paths with a configuration file specifying an option for specific file(s) and a pylint section, ignored warnings from the specific file(s) are not suppressed beginning in pylama 8.2.1

### ISSUE THREAD

**Issue by alangefe** [klen/pylama#208]
# Summary

All three of the following elements must be present to replicate this issue. If any one element is removed, the issue is no longer replicable:
1. Specify more than one path in the pylama arguments
2. Specify options for specific file(s) in the pylama options file
3. Specify a pylint options section in the pylama options file

When all three criteria are present, audit codes to ignore that are specified in the specific file section will no longer be honored.

This was working properly in pylama 8.2.0. It began failing in pylama 8.2.1 and is still failing in the latest (8.3.5).

# To replicate

a.py:

```
def method_a():
    pass
```

b.py:

```
def method_b(unused):
    pass
```

pylama.ini:

```
[pylama]
linters = pylint

[pylama:b.py]
ignore = W0613

[pylama:pylint]
good_names = f
```

When using pylama 8.2.0, the W0613 warning is correctly suppressed:

```
$ pylama --version
pylama 8.2.0
$ pylint --version
pylint 2.12.2
astroid 2.9.0
Python 3.8.10 (default, Sep 28 2021, 16:10:42) 
[GCC 9.3.0]
$ pylama --verbose --options pylama.ini a.py b.py
Try to read configuration from: 'pylama.ini'
Namespace(abspath=False, concurrent=False, file_params={re.compile('(?s:b\\.py)\\Z'): {'ignore': 'W0613'}}, format='pycodestyle', from_stdin=False, hook=False, ignore=set(), linters=['pylint'], linters_params={'pylint': {'good_names': 'f'}}, max_complexity=10, max_line_length=100, options='pylama.ini', paths=['a.py', 'b.py'], pydocstyle_convention=None, pylint_confidence=None, report=None, select=set(), skip=[], sort=None, verbose=True)
Run [pylint] a.py
Run [pylint] b.py
a.py:1:1 C0114 Missing module docstring [pylint]
a.py:1:1 C0116 Missing function or method docstring [pylint]
b.py:1:1 C0114 Missing module docstring [pylint]
b.py:1:1 C0116 Missing function or method docstring [pylint]
```

Beginning in pylama 8.2.1, the W0613 warning is emitted even though the ignore statement in pylama.ini indicates that it should be suppressed:

```
$ pylama --version
pylama 8.2.1
$ pylint --version
pylint 2.12.2
astroid 2.9.0
Python 3.8.10 (default, Sep 28 2021, 16:10:42) 
[GCC 9.3.0]
$ pylama --verbose --options pylama.ini a.py b.py
Try to read configuration from: 'pylama.ini'
Namespace(abspath=False, concurrent=False, file_params={re.compile('(?s:b\\.py)\\Z'): {'ignore': 'W0613'}}, format='pycodestyle', from_stdin=False, hook=False, ignore=set(), linters=['pylint'], linters_params={'pylint': {'good_names': 'f'}}, max_complexity=10, max_line_length=100, options='pylama.ini', paths=['a.py', 'b.py'], pydocstyle_convention=None, pylint_confidence=None, report=None, select=set(), skip=[], sort=None, verbose=True)
Run [pylint] a.py
Run [pylint] b.py
a.py:1:1 C0114 Missing module docstring [pylint]
a.py:1:1 C0116 Missing function or method docstring [pylint]
b.py:1:1 C0114 Missing module docstring [pylint]
b.py:1:1 C0116 Missing function or method docstring [pylint]
b.py:1:14 W0613 Unused argument 'unused' [pylint]
```

As I note above, the issue goes away if:
1. The first path, a.py, is removed.
2. The `[pylama:pylint]` section of pylama.ini is removed.

This may be the same issue as #205, but there were not enough details in that issue for me to be sure they were the same.

**Comment by klen** [comment]
@alangefe Thank you for such a detailed explanation. I've pushed the fix (8.3.1).

**Comment by alangefe** [comment]
@klen , I've verified that this is indeed fixed in pylama 8.3.6. Thanks for the quick fix!
